app/services/bot_checker.py

```diff
@@ -0,0 +1,53 @@
+import logging
+
+from fastapi import Depends
+
+from app.core.config import settings
+from app.models import Tweet, TwitterUser
+
+from .ml import ML
+from .scrape import TwitterScraper
+
+_logger = logging.getLogger(__name__)
+
+
+class BotChecker:
+    def __init__(self, ml: ML = Depends(), scraper: TwitterScraper = Depends()):
+        self._ml_service = ml
+        self._scraper = scraper
+
+    def check_account(self, username):
+        user_db: TwitterUser = TwitterUser.objects(username=username).first()
+        if user_db is None:
+            _logger.info("This account is not exist in DB")
+            user = self._scraper.get_user_by_username(username)
+            recent_tweets = [
+                Tweet(
+                    tweet_id=str(tweet.id), text=tweet.text, created_at=tweet.created_at
+                )
+                for tweet in self._scraper.get_tweet_info(
+                    user.id_str, settings.TWEETS_NUMBER
+                )
+            ]
+            user_db = TwitterUser(
+                twitter_id=user.id_str,
```

Comment:
When was this changed into (int, str)? I wasnt following

Reply:
What do you mean?

Reply:
I remember that the `id` field was string before right?

Reply:
`id` field of what?

Reply:
of `TwitterUser`

Reply:
But `TwitterUser.twitter_id` is always string?

Reply:
I saw `TwitterID = Union[int, str]` in `services/scrape.py`. Thought it's related to this.

Reply:
so what's `TwitterID = Union[int, str]` for then?

Reply:
Because we store Twitter ID as string in MongoDB, but Dung's model requires integer, and `tweepy` accept both string and integer